### DISCUSSION TITLE
Feature Request: Language Selection Module with a Focus on Multilingual Support

### DISCUSSION THREAD

**Discussion by Lugic1993** [immich-app/immich#8345] [General]
Dear Immich Team,

I hope this message finds you well. I am reaching out to express my immense admiration for the Immich platform. Your dedication to creating an exceptional user experience is evident through the intuitive design and functionality of the server, web, and mobile applications. It's clear that Immich stands out as a superior option compared to many alternatives available in China, thanks to its outstanding quality and user-centric approach.

However, there's one aspect that, if improved, could significantly enhance the accessibility and global reach of your application: language support. Currently, the application only operates in English, which, while comprehensive, limits the potential user base, especially in non-English speaking regions such as China. The addition of a language selection module, enabling users to switch seamlessly between languages, would be a monumental step forward.

Moreover, I am particularly interested in contributing to the translation efforts for Chinese, should you decide to expand the language options. I believe that with my assistance, along with your team's expertise, we can make Immich even more accessible and user-friendly for speakers of Chinese and beyond.

Thank you for your time and consideration. I look forward to your positive response and am excited about the possibility of contributing to the Immich community in a meaningful way.

Warm regards,

**Comment by mertalev** [comment]
We're aware of this and do want to support different languages for the web app, similar to the mobile app. It hasn't been prioritized yet, but a framework for translations will be added eventually.

I think the first step will be to move the current English texts to separate files and reference them by label.

**Comment by bo0tzz** [comment]
#1688
